ebean-test/src/test/resources/ebean.properties

```diff
@@ -72,7 +72,7 @@ datasource.db.databaseDriver=org.h2.Driver
 
 datasource.h2.username=sa
 datasource.h2.password=
-datasource.h2.url=jdbc:h2:mem:testsMem;DB_CLOSE_ON_EXIT=FALSE
+datasource.h2.url=jdbc:h2:mem:testsMem;DB_CLOSE_ON_EXIT=FALSE;NON_KEYWORDS=KEY,VALUE
```

Comment:
interesting workaround :)
Would you add some comments, when this is needed (maybe in documentation for elementCollection)